pallets/subtensor/src/lib.rs

```diff
@@ -780,7 +780,7 @@ pub mod pallet {
     pub type HotkeyEmissionTempo<T> =
         StorageValue<_, u64, ValueQuery, DefaultHotkeyEmissionTempo<T>>;
     #[pallet::storage]
-    /// Map ( hot ) --> emission | Accumulated hotkey emission.
+    /// Name corrected: PendingdHotkeyEmission => PendingHotkeyEmission
     pub type PendingdHotkeyEmission<T: Config> = StorageMap<
```

Comment:
why keep the the storage with the wrong name? I guess there is raw io interface can be used to iterate all data. I will check.